erpnext/accounts/notification/notification_for_new_fiscal_year/notification_for_new_fiscal_year.json

```diff
@@ -11,19 +11,21 @@
  "event": "New",
  "idx": 0,
  "is_standard": 1,
- "message": "<h3>{{_(\"Fiscal Year\")}}</h3>\n\n<p>{{ _(\"New fiscal year created :- \") }} {{ doc.name }}</p>",
- "modified": "2018-04-25 14:30:38.588534",
+ "message_type": "HTML",
+ "modified": "2023-11-17 08:54:51.532104",
  "modified_by": "Administrator",
  "module": "Accounts",
  "name": "Notification for new fiscal year",
  "owner": "Administrator",
  "recipients": [
   {
-   "email_by_role": "Accounts User"
+   "receiver_by_document_field": "Accounts User"
   },
   {
-   "email_by_role": "Accounts Manager"
+   "receiver_by_document_field": "Accounts Manager"
```

Comment:
These don't seem like document fields to me 🤔

Reply:
It's only in js:
```console
frappe/email/doctype/notification/notification.js
72-				});
73-			}
74-
75-			// set email recipient options
76-			frm.fields_dict.recipients.grid.update_docfield_property(
77:				"receiver_by_document_field",
78-				"options",
79-				[""].concat(["owner"]).concat(receiver_fields)
80-			);
81-		});
82-	},
```

Reply:
I didn't get what you're saying. My point was that `receiver_by_document_field` should most likely contain a valid field name, not a role. For roles we have `receiver_by_role`.

Reply:
Ah damn! Misunderstanding slumbering here. :smile: You are absolutely right. :+1: